docs/getting-started/installation.md

```diff
@@ -251,3 +251,18 @@ podAnnotations: {}
 [ecr]: https://gallery.ecr.aws/aquasecurity/trivy
 [registry]: https://github.com/orgs/aquasecurity/packages/container/package/trivy
 [helm]: https://github.com/aquasecurity/trivy/tree/{{ git.tag }}/helm/trivy
+
+
+## pre-commit hook
+
+Add this to your [pre-commit](https://pre-commit.com/) `.pre-commit-config.yaml` config:
```

Comment:
can we move this and another logic into Makefile?
- `pre-commit` doesn't have install script, but we can add check that `pre-commit` is installed, as for [tiny-go](https://github.com/aquasecurity/trivy/blob/ed322ac6664306c13ed673e0fa7dd6f76ebc9841/Makefile#L53-L59).
- we can add script to create `.pre-commit-config.yaml` file(if file doesn't exist) and install this config.